jetstream/consumer.go

```diff
@@ -317,6 +318,42 @@ func deleteConsumer(ctx context.Context, js *jetStream, stream, consumer string)
 	return nil
 }
 
+func pauseConsumer(ctx context.Context, js *jetStream, stream, consumer string, pauseUntil *time.Time) error {
+	ctx, cancel := wrapContextWithoutDeadline(ctx)
+	if cancel != nil {
+		defer cancel()
+	}
+	if err := validateConsumerName(consumer); err != nil {
+		return err
+	}
+	subject := apiSubj(js.apiPrefix, fmt.Sprintf(apiConsumerPauseT, stream, consumer))
+
+	var resp consumerPauseResponse
+	req, err := json.Marshal(consumerPauseRequest{
+		PauseUntil: pauseUntil,
+	})
```

Comment:
```suggestion
	req, err := json.Marshal(consumerPauseRequest{
		PauseUntil: nil,
	})
```

Should this work with `nil` PauseUntil value?

It didn't work for me; I had to change to a `time.Now()` value instead.

Should I send a empty body when I want to `resume` right away?

Need help here!